theano/tensor/tests/test_opt.py

```diff
@@ -6498,6 +6498,32 @@ def test_local_merge_alloc():
     assert_raises((AssertionError, ValueError), f, 0., 1, 2, 5, 3, 4)
 
 
+def test_local_log_sum_exp1():
+    # Tests if optimization is applied
+    x = matrix('x')
+    y = T.log(T.sum(T.exp(x), axis=1))
+    MODE = theano.compile.get_default_mode().including('local_log_sum_exp')
+    f = function([x], y, mode=MODE)
+
+    assert (f.maker.fgraph.toposort()[1].op.scalar_op ==
```

Comment:
Doing a list comprehension to verify that there is at least one `maximum`might be more robust to future changes.